tests/fuzzer.c

```diff
@@ -490,6 +490,24 @@ static int basicUnitTests(U32 const seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    DISPLAYLEVEL(3, "test%3i :  : ldm fill dict out-of-bounds check", testNb++);
```

Comment:
If this is passing, then this test doesn't expose the bug, since it isn't fixed.